client/src/js/filters/filters.js

```diff
@@ -1,109 +1,111 @@
+  // passing through individual locale and doesn't expose utility methods 
+  .filter('intlcurrency', [
+    '$http',
+    '$filter',
+    '$sce',
+    'store',
+    'messenger',
+    function ($http, $filter, $sce, Store, messenger) {
+
+      var currency = new Store({
+        data : [],
+        identifier : 'id'
+      });
 
-(function (angular) {
-  'use strict';
+      // $http.get('/finance/currencies')
+      // .success(function (data) {
+        // currency.setData(data);
+      // })
+      // .error(function (error) {
+        // messenger.danger('An error occured:' + JSON.stringify(error));
+      // });
 
-  angular.module('bhima.filters', [])
-    .filter('boolean', function() {
-      return function (input) {
-        return Boolean(Number(input));
-      };
-    })
-
-    .filter('intlcurrency', [
-      '$http',
-      '$filter',
-      '$sce',
-      'store',
-      'messenger',
-      function ($http, $filter, $sce, Store, messenger) {
-
-        var currency = new Store({
-          data : [],
-          identifier : 'id'
-        });
+      return function (value, id) {
+        value = (value || 0).toFixed(2);
 
-        $http.get('/finance/currencies')
-        .success(function (data) {
-          currency.setData(data);
-        })
-        .error(function (error) {
-          messenger.danger('An error occured:' + JSON.stringify(error));
-        });
-
-        return function (value, id) {
-          value = (value || 0).toFixed(2);
+        if (!angular.isDefined(id) || !angular.isDefined(currency)) {
+          return $sce.trustAsHtml($filter('currency')(value));
+        }
 
-          if (!angular.isDefined(id) || !angular.isDefined(currency)) {
-            return $sce.trustAsHtml($filter('currency')(value));
-          }
+        // first, extract the decimal digits '0.xx'
+        var decimalDigits = value.slice(value.indexOf('.')+1, value.indexOf('.') + 3);
 
-          // first, extract the decimal digits '0.xx'
-          var decimalDigits = value.slice(value.indexOf('.')+1, value.indexOf('.') + 3);
+        if (decimalDigits) {
+          value = value.slice(0, value.indexOf('.'));
+        }
 
-          if (decimalDigits) {
-            value = value.slice(0, value.indexOf('.'));
-          }
+        // var templ = value.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1'+currency.get(id).separator);
+        // templ += '<span class="desc">' + currency.get(id).decimal + decimalDigits + '</span><span class="cur"> ' + currency.get(id).symbol +  '</span>';
 
-          var templ = value.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1'+currency.get(id).separator);
-          templ += '<span class="desc">' + currency.get(id).decimal + decimalDigits + '</span><span class="cur"> ' + currency.get(id).symbol +  '</span>';
+        // return $sce.trustAsHtml(templ);
+        return $sce.trustAsHtml('<span style="color : red;">Filter Depriciated</span>');
```

Comment:
Welp.  I should have read more code.  Ignore [the comment above](https://github.com/IMA-WorldHealth/bhima/pull/689/files#r38726482).